commitlint.config.ts

```diff
@@ -76,6 +76,14 @@ function isFooterNote(line: string): boolean {
         isFixesSentence(line);
 }
 
+function includesHashtagRef(text: string) {
+    return text.match(`#[0-9]+`) !== null;
+}
+
+function removeAllCodeBlocks(text: string) {
+    return text.replace(/```[^]*```/g, ''); // maybe I should include \n in the pattern
```

Comment:
I'm confused by this comment; why don't you do some testing to figure out if \n should be included or not in the pattern?

Reply:
Good point!

Reply:
I removed the comment because I didn't find a failing case